doc-Appliance_Hardening_Guide/miq/build/en-US/master.xml

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE book [
 <!ENTITY % sgml.features "IGNORE">
 <!ENTITY % xml.features "INCLUDE">
-<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/dbcentx.mod">
+<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.6//EN" "http://www.oasis-open.org/docbook/xml/4.6/dbcentx.mod">
```

Comment:
Hi @adahms - looks like the find and replace picked up the Docbook version as well -- should this remain as 4.5?